certbot/tests/util.py

```diff
@@ -361,56 +361,44 @@ def setUp(self):
         self.config.server = "https://example.com"
 
 
+def _handle_lock(queue_in, queue_out, path):
```

Comment:
On current implementation, using queue is overkill compared to queue. It was because before, I was checking for specific signal values during the coordination (START/LOCKED/STOP), but it was also overkill for a test method.

I move to an event, and add back docstring.